app/main.py

```diff
@@ -2,4 +2,55 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
+
 # write your code here
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
```

Comment:
Specify inner type of list. Also, if argument ca be None, you must show in type hint
```suggestion
    def __init__(self, name: str, weight: int, coords: list[int] | None = None) -> None:
```